poly/src/domain/utils.rs

```diff
@@ -37,13 +38,14 @@ pub(crate) fn compute_powers_and_mul_by_const_serial<F: Field>(
         .collect()
 }
 
+#[allow(unused)]
```

Comment:
Putting allow(unused) here, since we want to benchmarking changing roots of unity with this function